### PR TITLE
Ensure Prettier passes in CI

### PR DESCRIPTION
**Pull request by rob-mccann**
This ensures prettier runs and passes in CI before marking a PR as ✅ 

Also conforms existing code to prettier's rules.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-Apache License
-==============
+# Apache License
```
Comment (balloob) [inline LICENSE.md:1]: Don't prettier the license.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -17,23 +17,23 @@ diverse, inclusive, and healthy community.
 Examples of behavior that contributes to a positive environment for our
 community include:
 
-* Demonstrating empathy and kindness toward other people
-* Being respectful of differing opinions, viewpoints, and experiences
-* Giving and gracefully accepting constructive feedback
-* Accepting responsibility and apologizing to those affected by our mistakes,
+- Demonstrating empathy and kindness toward other people
```
Comment (balloob) [inline CODE_OF_CONDUCT.md:20]: Exclude our CoC. It's the same across repo's.

File: CLA.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ By making a contribution to this project, I certify that:
```
Comment (balloob) [inline CLA.md:25]: Exclude CLA.

File: LICENSE.md (hunk 1 above)
Comment (rob-mccann) [reply LICENSE.md]: Done :+1:

**Review by balloob — APPROVED** [review]

**Pull request merged**